model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/EntityMemberDeclarationTests.xtend

```diff
@@ -181,4 +181,38 @@ class EntityMemberDeclarationTests {
             error
         )
     }
+    
+    @Test
+    def void testQueryParameterExpression() {
+        '''
+			model test;
+
+			import judo::types;
+
+			entity E {
+				field Integer e;
+				
+				query Integer q(Integer p = 10 + 10) => E!all()!size();
```

Comment:
What is this? Is there parameter which is not used? Provide tests which resolves this substitution.